tests/installation/bootloader_svirt.pm

```diff
@@ -40,6 +40,12 @@ sub run() {
         $xenconsole = "xvc0";
     }
 
+    # The root of all problems is this: Xen closes VNC and serial console connections
+    # on reboot. Unlike KVM. So, to know when we are restarting if we are in the
+    # state before, or after restart we have to configure libvirt to destroy
+    # (i.e. turn off) the VM. Then we have to explicitely start it define_and_start.
+    $svirt->change_domain_element(on_reboot => 'destroy') if ($vmm_family eq 'xen');
```

Comment:
Makes sense. I created https://github.com/os-autoinst/os-autoinst/pull/796. But still I want to keep this provision until the os-autoinst change is deployed, otherwise it cripples Xen testing, I'll remove it afterwards.